test/LibraryManager.IntegrationTest/LibmanCompletionTests.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System.Collections.Generic;
 using Microsoft.Test.Apex.Editor;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NuGet;
```

Comment:
>NuGet [](http://example.com/codeflow?start=6&length=5)

Is this needed? Did namespace of the used classes changed?

Reply:
It is not (there was an error while I was updating the .vsix project's csproj file). On reopen, I have verified that the namespace for "SemanticVersion" is Microsoft.Web.LibraryManager.  Undone.